src/pages/components/dots.mdx

```diff
@@ -84,6 +84,9 @@ import SizeCode from '!!raw-loader!../../examples/loaders/dots/DotsSize.tsx';
 
 <Component components={props.data.mdx.components} componentName="dots" />
 
+Nest a Dots within interactive elements or components for a users
+action that is in flight.
```

Comment:
Historically, we've refrained from spelling out design guidance (or repeating info already delineated under "Used for this"/"Not for this") under the API sections. Has content guidance changed?

Reply:
"...within interactive elements" would be considered design guidance (vs. "you can place a `Dots` loader anywhere you wish"), usually reserved for documentation outside of the API section.

Reply:
This also felt out of place for me. When I was working with Smriti on her API documentation sentences that talked about nesting was about how to place specific things within each other correctly: see [accordion](https://garden.zendesk.com/components/accordion).

Is this because Dots could be nested in many types of components...? I don't know if we're using the nesting content the same here as we did in Accordion.

Reply:
Understood, that makes sense!